scripts/tools/nrfconnect/nrfconnect_factory_data.schema

```diff
@@ -156,6 +156,32 @@
             "minLength": 36, 
             "maxLength": 36
         },
+        "appearance": {
+            "description": "Product appearance provides a description of the product's finish method (Matte, Satin, Polished etc.) and products primary color, saturation and hue",
```

Comment:
```suggestion
            "description": "Product appearance provides a description of the product's external case finish method (matte, satin, polished) and the product's primary color, saturation, and hue",
```
Do we need the "etc" in the parentheses if you only mention 3 values in the description of `finish` (lines 161-163)?